brd-ios/breadwallet/src/ViewControllers/KYC/KYCAddress/Cells/KYCAddressFieldsView.swift

```diff
@@ -98,6 +98,7 @@ class KYCAddressFieldsView: BaseView, GenericSettable {
     override func setupSubviews() {
         super.setupSubviews()
         
+        backgroundColor = Theme.primaryBackground
```

Comment:
please, have an empty line after this or move this to end of the `setupSubviews` function.